app/lib/controlled_work_mappings/civ_means_7.yml

```diff
@@ -52,6 +52,7 @@
       source: smod_assets?
       yes_value: "Yes"
       no_value: "No"
+
```

Comment:
are these empty lines just to split the sections visually?

Reply:
I think this is an artefact of changing the name i.e. removing the suffx `_new` and deleting the previous file which had the same name.

Reply:
thank you